adhocteam/Head-Start-TTADP
# [TTAHUB-279] implement redesigned filter & sorting for the topics bar graph

## Description of change

Implemented new design for the specialists filter (now a set of checkboxes) and the sorting (the options now match the original design, and the apply filters button is within the dropdown menu).

## How to test

Visit the dashboard. Scroll to the bar graph at the bottom. Observe that the controls for sorting and filtering by specialist are different - the apply filters is confined to each box now. Test them out. Watch the data filter and the graph redraw.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-279

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
